ee/clickhouse/queries/funnels/funnel_event_query.py

```diff
@@ -11,13 +14,23 @@ def get_query(self, entities=None, entity_name="events", skip_entity_filter=Fals
             f"{self.EVENT_TABLE_ALIAS}.event as event, {self.EVENT_TABLE_ALIAS}.team_id as team_id, {self.EVENT_TABLE_ALIAS}.distinct_id as distinct_id, {self.EVENT_TABLE_ALIAS}.timestamp as timestamp, {self.EVENT_TABLE_ALIAS}.properties as properties, {self.EVENT_TABLE_ALIAS}.elements_chain as elements_chain"
             + (f", {self.DISTINCT_ID_TABLE_ALIAS}.person_id as person_id" if self._should_join_distinct_ids else "")
             + (f", {self.PERSON_TABLE_ALIAS}.person_props as person_props" if self._should_join_persons else "")
+            + (
+                " ".join(
+                    [
+                        f", {self.EVENT_TABLE_ALIAS}.properties_{prop} as properties_{prop}"
+                        for prop in settings.CLICKHOUSE_DENORMALIZED_PROPERTIES
+                    ]
+                )
+                if settings.CLICKHOUSE_DENORMALIZED_PROPERTIES
+                else ""
+            )
         )
 
         date_query, date_params = self._get_date_filter()
         self.params.update(date_params)
 
         prop_filters = self._filter.properties
-        prop_query, prop_params = self._get_props(prop_filters)
+        prop_query, prop_params = self._get_props(prop_filters, allow_denormalized_props=True)
```

Comment:
this should be governed by `settings.CLICKHOUSE_DENORMALIZED_PROPERTIES` too?

Reply:
If you look at `prop_filter_json_extract` there's a check for that already. This just shows that the query we're about to make works with denormalised props